### PR TITLE
Second iteration of policy for resetting second factor for cloud.gov IDP

### PR DESCRIPTION
**Pull request by brittag**
This is a followup to [this earlier PR about MFA reset policies](https://github.com/18F/cg-site/pull/866), providing a simpler procedure.

This is based on [the additional discussion here](https://docs.google.com/document/d/1LvPb33vmZGA7zWEpFevWhBv_76nJCXMXe4jqFFX-TsY/edit#heading=h.afmvcdkdd868).

Similarly, requesting review from both @rogeruiz as Cloud Ops representative and @khandelwal as System Owner; we shouldn't merge this without approval from @khandelwal.

### CODE REVIEW

File: content/docs/getting-started/accounts.md
Comment (khandelwal) [inline content/docs/getting-started/accounts.md]: This first sentence might need a little clarity. Are we defining multi-factor authentication, or are we defining what MFA means in the cloud.gov context?

Most of our audience will be familiar with MFA (and increasingly so) as time goes on.

Comment (khandelwal) [inline content/docs/getting-started/accounts.md]: "such as Google Authenticator ..." doesn't make it crystal clear that those are the two choices. I'm assuming users aren't free to choose which MFA application they want.

File: content/docs/getting-started/accounts.md
Comment (khandelwal) [inline content/docs/getting-started/accounts.md]: Putting a title of sorts above this section might help readability. Something like "What happens if I lose my token?"

File: content/docs/getting-started/accounts.md
Comment (khandelwal) [inline content/docs/getting-started/accounts.md]: I think this is the first time we mention the phrase 'second factor'. Do we need to clarify what this means?

**Review by khandelwal — DISMISSED** [review]

**Review by rogeruiz — COMMENTED** [review]
It looks good from a technical perspective, based on what we covered yesterday after standup. I'll defer to @khandelwal's review for the final approval.

File: content/docs/getting-started/accounts.md
Comment (rogeruiz) [reply content/docs/getting-started/accounts.md]: We had this explanation in our documentation before. The above defines MFA as what it means in both contexts. Though it might make sense to consider leveraging the same words we use on the login screens from 18F/Shibboleth-IdP3-TOTP-Auth#15. e.g. `MFA app` and `token code`.

Comment (rogeruiz) [reply content/docs/getting-started/accounts.md]: It might make sense to stretch this out to include any [Time-based One-time Password Algorithm](https://en.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm) (e.g.`TOTP`) application.  We had this question come in through support because we recommend `smartphone apps` and they wanted to use `a Google Chrome extension`. They're free to use any TOTP application and there are several Desktop applications as well. [Authy has most platforms covered](https://authy.com/features/multiple-devices/). 

Maybe we can just call it _something you know (your password)_ and _something you have (an MFA/2FA application)_? or do we need to keep the phone part as part of the SSP?

File: content/docs/getting-started/accounts.md
Comment (rogeruiz) [reply content/docs/getting-started/accounts.md]: Meh, I'd say it's similar enough to  multi-factor / second-factor. Maybe when we explain MFA we can say `first factor ( password )` and `second factor ( MFA/2FA app )` ?? 🤷‍♂️

**Comment by brittag** [comment]
@rogeruiz I added a new commit with language adjustments to address Shashank's comments - up for taking another look? If you think it's ok, we can merge it.

**Review by rogeruiz — APPROVED** [review]
looks great! :shipit:

File: content/docs/getting-started/accounts.md
Comment (rogeruiz) [inline content/docs/getting-started/accounts.md]: Good catch. Are we waiting on @khandelwal approval? or should I dismiss his previous one since it looks like you've addressed everything. wdyt? @brittag

Comment (brittag) [reply content/docs/getting-started/accounts.md]: @rogeruiz Go ahead and dismiss them - I approve this in his place! We don't have to wait for him to return from OOO; I have confidence we have addressed his suggestions.

**Pull request merged**
